### ISSUE TITLE
Workaround for keyspace-problem?

### ISSUE THREAD

**Issue by realSnoopy** [hashcat/hashcat#2736]
regarding to this
https://hashcat.net/wiki/doku.php?id=frequently_asked_questions#what_is_a_keyspace
`In other words, hashcat's --keyspace is specifically designed to optimize distribution of work, and is not a literal representation of the total possible keyspace for a given attack.`
OKAY got it
but 
in mode -a 3 it says
`-a 3 – multiply the number of possible characters in each mask position for the base loop mask, excluding the mod loop mask – please just use --keyspace switch ;)`
okay this is the reason for these numbers

```
mp64.exe --combinations ?a -> 95
hashcat --keyspace -a 3 ?a -> 1
mp64.exe --combinations ?a?a?a?a?a?a?a?a -> 6634204312890625
hashcat --keyspace -a 3 ?a?a?a?a?a?a?a?a -> 7737809375
```
so basically in -a 3 the keyspace should always? be smaller than the actual combinations, because of the excluded mod loop

BUT why maskprocessor is capable of doing things like
```
mp64.exe --combinations ?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a?a (24*?a)
11007294638326990593
BUT
hahscat fails on
hashcat --keyspace -a 3 ?a?a?a?a?a?a?a?a?a?a (10*?a)
Integer overflow detected in keyspace of mask: ?a?a?a?a?a?a?a?a?a?a
```
yeah i/we can feed hashcat from stdin with maskprocessor but is there a workaround to bypass this keyspace calculation problem?

**Comment by jsteube** [comment]
No that's a misunderstanding. The mp64.exe is not able to do it. It just misses the check for overflow that hashcat actually has.
